arcade/examples/sprite_health.py

```diff
@@ -1,230 +1,249 @@
+        if self.player_sprite.health <= 0:
+            arcade.exit()
 
-    def on_mouse_motion(self, x, y, dx, dy):
-        """
-        Called whenever the mouse moves.
-        """
-        self.player_sprite.center_x = x
+        # Increase the enemy's timer
+        self.enemy_timer += delta_time
 
-    def on_mouse_press(self, x, y, button, modifiers):
-        """
-        Called whenever the mouse button is clicked.
-        """
-        # Gunshot sound
-        arcade.play_sound(self.gun_sound)
-        # Create a bullet
-        bullet = arcade.Sprite(":resources:images/space_shooter/laserBlue01.png", SPRITE_SCALING_LASER)
+        # Update the player's indicator bar position
+        self.player_sprite.indicator_bar.position = self.player_sprite.center_x, self.player_sprite.center_y + INDICATOR_BAR_OFFSET
 
-        # The image points to the right, and we want it to point up. So
-        # rotate it.
-        bullet.angle = 90
+        # Call updates on bullet sprites
+        self.bullet_list.update()
 
-        # Give the bullet a speed
-        bullet.change_y = BULLET_SPEED
+        # Check if the enemy can attack. If so, shoot a bullet from the enemy to the player
+        if self.enemy_timer >= ENEMY_ATTACK_COOLDOWN:
+            self.enemy_timer = 0
 
-        # Position the bullet
-        bullet.center_x = self.player_sprite.center_x
-        bullet.bottom = self.player_sprite.top
+            # Create the bullet
+            bullet = arcade.Sprite(":resources:images/space_shooter/laserBlue01.png", SPRITE_SCALING_BULLET)
 
-        # Add the bullet to the appropriate lists
-        self.bullet_list.append(bullet)
+            # Set the bullet's position
+            bullet.position = self.enemy_sprite.position
 
-    def on_update(self, delta_time):
-        """ Movement and game logic """
+            # Set the bullet's angle to face the player
+            diff_x = self.player_sprite.center_x - self.enemy_sprite.center_x
+            diff_y = self.player_sprite.center_y - self.enemy_sprite.center_y
+            angle = math.atan2(diff_y, diff_x)
+            angle_deg = math.degrees(angle)
+            if angle_deg < 0:
+                angle_deg += 360
+            bullet.angle = angle_deg
 
-        # Call update on bullet sprites
-        self.bullet_list.update()
+            # Give the bullet a velocity towards the player
+            bullet.change_x = math.cos(angle) * BULLET_SPEED
+            bullet.change_y = math.sin(angle) * BULLET_SPEED
+
+            # Add the bullet to the bullet list
+            self.bullet_list.append(bullet)
 
         # Loop through each bullet
         for bullet in self.bullet_list:
-
-            # Check this bullet to see if it hit a coin
-            hit_list = arcade.check_for_collision_with_list(bullet, self.coin_list)
-
-            # If it did, get rid of the bullet
-            if len(hit_list) > 0:
+            # Check if the bullet has gone off-screen. If so, kill it
```

Comment:
This comment is low priority if you're crunched for time.

I know we have kill as an alias for `remove_from_sprite_lists`, but I think that phrasing might confuse new players. Maybe delete would be better since it's not alive?